awx/ui_next/SEARCH.md

```diff
@@ -24,143 +49,256 @@
 - DONE add search filter removal test for qs.
 - DONE remove button for search tags of duplicate keys are broken, fix that
 
-## TODO later on in 3.6: stuff to be finished for search iteration 1 (I'll card up an issue to tackle this. I plan on doing this after I finished the awx project branch work)
+### TODO pre-holiday break
+- Update COLUMNS to SORT_COLUMNS and SEARCH_COLUMNS
+- Update to using new PF Toolbar component (currently an experimental component)
+- Change the right-hand input based on the type of key selected on the left-hand side.  In addition to text input, for our MVP we will support:
+  - number input
+  - select input (multiple-choice configured from UI or Options)
+- Update the following lists to have the following keys:
+
+**Jobs list** (signed off earlier in chat)
+  - Name (which is also the name of the job template) - search is ?name=jt
+  - Job ID - search is ?id=13
+  - Label name - search is ?labels__name=foo
+  - Job type (dropdown on right with the different types) ?type = job
```

Comment:
is this intended to *not* be a substring search? (`job` is a substring in all job types, which makes it not useful as a distinguisher)

Reply:
@wenottingham this will be a multiple choice selection (not a substring).  the options will be:

Text (string) inputs are substring/case-insensitive (we append __icontains to all those keys).

```
"choices": [
                    [
                        "project_update",
                        "SCM Update"
                    ],
                    [
                        "inventory_update",
                        "Inventory Sync"
                    ],
                    [
                        "job",
                        "Playbook Run"
                    ],
                    [
                        "ad_hoc_command",
                        "Command"
                    ],
                    [
                        "system_job",
                        "Management Job"
                    ],
                    [
                        "workflow_job",
                        "Workflow Job"
                    ]
                ]
```